terraform/iam_role_policy.tf

```diff
@@ -160,3 +160,33 @@ resource "aws_iam_role_policy" "ecs_grafana_task_cloudwatch_read" {
 
   policy = "${data.aws_iam_policy_document.allow_cloudwatch_read_metrics.json}"
 }
+
+resource "aws_iam_role_policy" "drain_ecs_container_instance_sns" {
+  name   = "drain_ecs_container_instance_publish_to_sns"
+  role   = "${module.lambda_drain_ecs_container_instance.role_name}"
+  policy = "${module.ec2_terminating_topic.publish_policy}"
+}
+
+resource "aws_iam_role_policy" "drain_ecs_container_instance_asg_complete" {
+  name   = "drain_ecs_container_instance_asg_complete"
+  role   = "${module.lambda_drain_ecs_container_instance.role_name}"
+  policy = "${data.aws_iam_policy_document.complete_lifecycle_hook.json}"
+}
+
+resource "aws_iam_role_policy" "drain_ecs_container_instance_ecs_list" {
+  name   = "drain_ecs_container_instance_ecs_list"
+  role   = "${module.lambda_drain_ecs_container_instance.role_name}"
+  policy = "${data.aws_iam_policy_document.ecs_list_container_tasks.json}"
+}
+
+resource "aws_iam_role_policy" "drain_ecs_container_instance_ec2_describe" {
+  name   = "drain_ecs_container_instance_ec2_describe"
+  role   = "${module.lambda_drain_ecs_container_instance.role_name}"
+  policy = "${data.aws_iam_policy_document.ec2_describe_instances.json}"
+}
+
+resource "aws_iam_role_policy" "drain_ecs_container_send_asg_heartbeat" {
+  name   = "drain_ecs_container_instance_send_asg_heartbeat"
+  role   = "${module.lambda_drain_ecs_container_instance.role_name}"
+  policy = "${data.aws_iam_policy_document.send_asg_heartbeat.json}"
+}
```

Comment:
There are comments in the rest of this file to break it up, telling us where one set of role policies end and another begins. Could we get one here please?